_posts/en/newsletters/2020-10-14-newsletter.md

```diff
@@ -0,0 +1,180 @@
+  about revising the [BIP173][] rules for [bech32][topic bech32]
+  addresses in order to prevent the [bech32 extension bug][] from
+  affecting users of [taproot][topic taproot] and future similar
+  upgrades.  Russell proposed using a backwards-incompatible format
+  based on a [revised bech32 encoding scheme][wuille new bech32]
+  previously described by Pieter Wuille.  This would eliminate the bug
+  but also require wallets to upgrade in order to be able to pay taproot
+  users.
+
+    A [previously proposed alternative][news107 bech32] would be a
+    backwards-compatible restriction on bech32 address lengths.  This
+    only directly provides safety to taproot users receiving payments
+    from spenders who upgraded their wallets to enforce the new length
+    restrictions.  In the discussion, it was proposed that broader
+    safety could be provided by also enforcing length restrictions at
+    either the [consensus layer][harding bech32] or the [transaction
+    relay policy layer][o'connor bech32].
+
+    Russell concluded his post by noting that "the sooner a decision is
+    reached on this, the sooner we can begin upgrading software for a
+    taproot world."  Feedback would be especially appreciated from
+    authors of wallets and bitcoin-sending services as they will be
+    asked to implement whatever change is decided upon.
+
+- **Updated witness asymmetric payment channel proposal:** Lloyd
+  Fournier [posted][fournier update] to a thread from several weeks ago
+  about witness asymmetric payment channels (see [Newsletter
+  #113][news113 witasym]).  The idea behind this proposal is to change
+  how LN users get the evidence they need to prove their channel
+  counterparty cheated.  Currently the evidence is placed in separate
+  transaction for each channel participant; Fournier proposes placing
+  the evidence in separate signatures ("witnesses") for each
+  participant by using [signature adaptors][topic adaptor signatures].
+  Advantages of this approach are that it provides a protocol that
+  should be easier to integrate with hoped-for improvements to Bitcoin
+  (e.g. [schnorr signatures][topic schnorr signatures] with and without
+  [MuSig][topic musig] aggregation) and hoped-for improvements to LN
+  (e.g. a switch from [HTLCs][topic htlc] to [PTLCs][topic ptlc]).  The approach
+  may also be conceptually simpler, which could help attract more
+  security review of LN's basic operations.
+
+    This week, Fournier linked to an [updated version][fournier v2] of
+    his proposed protocol.  The main difference from the original
+    version is that a party who broadcasts a revoked signature reveals
+    the primary private key they use in the channel's multisig spends.
+    The other party to the channel can use that key combined with their
+    own key to immediately claim all funds in the channel.  Along
+    with another change, <!-- I think it also needs something like
```

Comment:
From the comment here looks like you are already investigating the details of "another change", which was to be my suggestion.